arch/arm64/configs/crocodile_defconfig

```diff
@@ -218,6 +218,7 @@ CONFIG_USB_SERIAL=m
 CONFIG_USB_SERIAL_CP210X=m
 CONFIG_USB_SERIAL_FTDI_SIO=m
 CONFIG_USB_SERIAL_OPTION=m
+CONFIG_USB_HUB_USB251XB=y
```

Comment:
Although, we have this hub on board, we don't use the driver but the default configuration of the hub.

Reply:
I2C lines to configure the hub are not connected -> building in the driver is pointless